src/SourceBuild/content/Directory.Build.targets

```diff
@@ -1,8 +1,11 @@
 <Project>
 
-  <Target Name="DetermineMicrosoftSourceBuildIntermediateInstallerVersion">
-    <!-- Manually load the installer version from the PVP.     -->
-    <XmlPeek XmlInputPath="$(IntermediatePath)PackageVersions.package-source-build.Current.props"
+  <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.Arcade.Sdk" Condition="'$(SkipArcadeSdkImport)' != 'true'" />
+
+  <Target Name="DetermineMicrosoftSourceBuildIntermediateInstallerVersion"
```

Comment:
I think it would be good eventually to split out the source-build specific targets and props into separate files rather than having them sprinkled together.

Reply:
Agreed. Initially I had this split and then added it back here as I believed that this would add too much noise to this change.